ui/src/views/404/NavigationItem/style.js.scss

```diff
@@ -0,0 +1,30 @@
+nav_item {
+    display: block;
+    padding: 10px 2.5%;
+    margin: 0 2.5%;
+    border-bottom: 1px solid #f4f4ee;
```

Comment:
I feel it would be great if we could extract all colors in the upcoming process, to collect them and then decide, if we can unify them a bit more. What do you think about putting this into a variables  (maybe even a `legacy-variables`) file?